packages/language/src/pli.langium

```diff
@@ -647,6 +642,17 @@ NumberLiteral: value=NUMBER;
 FQN returns string: ID ('.' ID)*;
 FeatureID returns string: ID;
 
+terminal CONCAT_EQUAL: '||=';
+terminal CONCAT_TOKEN: '||';
+terminal OR_EQUAL: '|=';
+terminal OR_TOKEN: '|';
+
+// TODO: Remove all ^ tokens once the lexer is working correctly
+terminal NOT_SMALLER: '¬<' | '^<';
+terminal NOT_EQUAL: '¬=' | '^=';
+terminal NOT_LARGER: '¬>' | '^>';
+terminal NOT_TOKEN: '¬' | '^';
+
```

Comment:
Do we consider adding composite symbol? Like: 
` (:` Start of type function parameter list  
`:)` End of type function parameter list

Reply:
I'm not quite sure I fully understand what you mean. Can you give an example?

Reply:
In Language Ref. In Composite symbols Table

<img width="560" alt="image" src="https://github.com/user-attachments/assets/75a7d7c7-d721-42ab-9a62-47b607e5bcd9" />

Reply:
I see. I don't think we support those at all right now (none of the examples have used that construct so far. However, this is unrelated to the compiler options, as the reason why we extract the `NOT` and `OR` tokens into dedicated terminals is because they can be configured. Meanwhile, the `:)` and (:` token types cannot be configured via the compiler options, so we don't need special handling for them.